packages/deployer/subtasks/clear-deployments.js

```diff
@@ -12,8 +11,6 @@ subtask(
   SUBTASK_CLEAR_DEPLOYMENTS,
   'Delete all previous deployment data on the current environment'
 ).setAction(async ({ instance }, hre) => {
-  await hre.run(TASK_CLEAN);
```

Comment:
The error was being caused because we were calling this Task programmatically from the deploy script on the same process, which removed the artifacts folders but it didn't remove the cache from hardhat, causing a missing error on the second+ run.

Also, this souldn't be necessary anymore as we started running the compile task with the `force: true` flag, which causes the regenartion of the artifacts.